website/content/docs/license/utilization/auto-reporting.mdx

```diff
@@ -213,8 +214,17 @@ reporting {
 
 The `development_cluster` configuration cannot differ across replicated clusters.
 In [performance replication](/vault/docs/enterprise/replication#performance-replication), secondary clusters inherit the
-`development_cluster` value from the primary cluster, regardless of their own HCL configuration. If the secondary cluster
-is promoted to a primary, it will begin to report its own `development_cluster` value determined by its HCL configuration.
+`development_cluster` value from the primary cluster, regardless of their own HCL configuration.
+
+<Note>
+
+If the inherited value from the primary differs from the HCL configuration on the secondary, the secondary will log the
+error `"development cluster setting in config does not match that of the primary, updating to follow the primary config setting"`
+to warn that it will be reporting a different value than what is set in its config.
+
+</Note>
```

Comment:
```suggestion
If the `development_cluster` setting in the secondary cluster configuration
differs from the value inherited from the primary cluster, the secondary cluster
logs the following error to warn about the discrepancy:

`"development cluster setting in config does not match that of the primary, updating to follow the primary config setting"`
```
Style correction: write in active voice, use asides sparingly (in this case, it works to just fold the information in to the main text)